projects/go-lib/styles/_forms.scss

```diff
@@ -1,10 +1,9 @@
-@import './variables';
-@import './mixins';
+@import "./variables";
+@import "./mixins";
 
-$input--disabled-background: rgba($base-dark-secondary, .4);
+$input--disabled-background: rgba($base-dark-secondary, 0.4);
```

Comment:
Across this repo, the decimal values has been beginning with ```.```. Let's keep consistent with practice.

Reply:
@jiaweicai92 would it be a good idea to switch practice on this? I know it's minor but I think having a 0 in front makes it a bit easier to read (and aesthetically pleasing, in my opinion). Like I said, this is a minor decision so I don't _really_ care but just a thought.

Reply:
@swimtech20, I personally prefer your and @adhiyan-tangoe's approach. Yet, let's keep the current format, for the sake of consistency. We could create a tech debt / followup task for refactoring the repo.

Reply:
@jiaweicai92 That sounds good to me.